### PR TITLE
Run git init on lerna init

### LINKED ISSUE #175: Lerna assumes the repo is already been initialized with `git init`.
I think if you lerna init in a folder that isn't a git repo already it errors? It's an edge case but we can run `git init` if it isn't

if we do this, we can remove the the line I added in the init section of the readme

### PR DESCRIPTION
**Pull request by kaicataldo**
fixes #175 

Thoughts on this approach?

### CODE REVIEW

File: src/Repository.js [hunk 1]
```diff
@@ -3,9 +3,16 @@ import FileSystemUtilities from "./FileSystemUtilities";
 import path from "path";
 
 export default class Repository {
-  constructor() {
-    this.rootPath = GitUtilities.getTopLevelDirectory();
```
Comment (kaicataldo) [inline src/Repository.js]: Error is thrown here

File: src/Command.js [hunk 2]
```diff
@@ -13,9 +13,9 @@ export default class Command {
     this.flags = flags;
 
     this.lernaVersion = require("../package.json").version;
-    this.repository = new Repository();
-    this.progressBar = progressBar;
     this.logger = logger;
+    this.repository = new Repository(this.logger);
```
Comment (kaicataldo) [inline src/Command.js]: Passing in the `logger` so we can log when initializing the Git repo

File: src/Repository.js (hunk 1 above)
Comment (hzoo) [inline src/Repository.js]: Don't think we have any tests for Repository yet

**Comment by hzoo** [comment]
Awesome!

File: src/GitUtilities.js [hunk 3]
```diff
@@ -3,6 +3,11 @@ import logger from "./logger";
 
 export default class GitUtilities {
   @logger.logifySync
+  static checkIfInitialized() {
```
Comment (thejameskyle) [inline src/GitUtilities.js]: The try-catch should happen in this util. It should be:

```js
static isInitialized() {
  try {
    ChildProcessUtilities.execSync("git rev-parse");
    return true;
  } catch (err) {
    return false;
  }
}
```

**Comment by thejameskyle** [comment]
#190

File: src/GitUtilities.js (hunk 3 above)
Comment (kaicataldo) [inline src/GitUtilities.js]: That makes sense - thanks for the explanation!

**Pull request merged**